src/frogger.cpp

```diff
@@ -51,10 +51,23 @@ Rcpp::List simulate_model(const leapfrog::StateSpace<ModelVariant> ss,
                           const int proj_years,
                           const int hiv_steps,
                           const std::vector<int> save_steps) {
+
+  int projection_period_int = 0;
+  Rcpp::String projection_period_str  = data["projection_period"];
+  if (projection_period_str  == "midyear") {
+    projection_period_int = 0;
+  } else if (projection_period_str  == "calendar") {
+    projection_period_int = 1;
```

Comment:
same suggestion to switch calendar to 0

Reply:
In 952c2a8 I have replaced this with calling the macro `leapfrog::internal::PROJPERIOD_CALENDAR`